server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -17,6 +17,24 @@ Feature: GetMessages method
     And the id of the message is "m1"
     And the message is in "custom,INBOX" mailboxes
 
+  Scenario: Retrieving a message in a mailbox delegated to me
+    Given "alice@domain.tld" has a mailbox "shared"
+    Given "alice@domain.tld" shares its mailbox "shared" with "bob@domain.tld"
+    And the user has a message "m1" in "shared" mailbox with subject "my test subject", content "testmail"
+    Given "bob@domain.tld" is connected
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 1 message
+    And the id of the message is "m1"
+
+  Scenario: Retrieving a message in a mailbox not delegated to me
+    Given "alice@domain.tld" has a mailbox "shared"
```

Comment:
what about naming it notshared?
what about adding a test where the mailbox is shared, but with another user (let's name here Carol)

Reply:
PS: if one of these tests is failing as stated in this commit, instead put it in ignore (that's a tag)

Reply:
About the order: It's github ordering: the branch introduce integretion tests at last (but they were 1st thing developed)